config/remotePlugin.js

```diff
@@ -97,6 +98,32 @@ const hacbs = {
         required: ['HACBS'],
       },
     },
+    {
+      type: 'console.page/route',
+      properties: {
+        path: '/app-studio/:appName/commit/:commitName',
+        exact: true,
+        component: {
+          $codeRef: 'CommitsPage',
+        },
+      },
+      flags: {
+        required: ['HACBS'],
```

Comment:
Please ensure hacbs pages also include the `SIGNUP` flag

Reply:
updated